lib/Workflow/Action.pm

```diff
@@ -173,18 +173,22 @@ This documentation describes version 2.05 of this package
 
 =head1 SYNOPSIS
 
- # Configure the Action...
- <action name="CreateUser"
-         class="MyApp::Action::CreateUser">
-   <field name="username" is_required="yes"/>
-   <field name="email" is_required="yes"/>
-   <validator name="IsUniqueUser">
-       <arg>$username</arg>
-   </validator>
-   <validator name="IsValidEmail">
-       <arg>$email</arg>
-   </validator>
- </action>
+ # Configure the action... (in e.g., 'actions.yaml')
+ action:
+ - name: CreateUser
+   class: MyApp::Action::CreateUser
+   field:
```

Comment:
Same comment as with the validator parameters above. Would have been much nicer to have;

```yaml
   username:
      is_required: yes
   email:
     is_required: yes
```